sample/src/main/java/com/patrykandpatryk/vico/sample/ui/ChartPager.kt

```diff
@@ -94,7 +98,12 @@ internal fun ChartPager(
                 SwipeHint(
                     currentPage = state.currentValue + 1,
                     pageCount = itemCount,
-                )
+                ) { direction ->
+                    val newState = (state.currentValue + direction)
```

Comment:
These parentheses seem to be unnecessary.

```suggestion
                    val newState = state.currentValue + direction
```